.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -34,7 +34,7 @@ body:
     id: reproduce
     attributes:
       label: Steps to Reproduce
-      description: Tell us how to reproduce this behaviour. Ideally, this should take the form of a [Minimum Workable Example](https://stackoverflow.com/help/minimal-reproducible-example)
+      description: Tell us how to reproduce this behaviour. Ideally, this should include a code block which produces the error.
```

Comment:
A code block is not necessarily always an MWE.

Reply:
This is the only comment that is a blocker from my end.

Reply:
I opted to do it this way because explaining what a MWE is was quite wordy, so instead of even mentioning that, I just switched it to asking for them to put in code that produces the error.

Reply:
Perhaps could we add the link to Wikipedia – https://en.wikipedia.org/wiki/Minimal_reproducible_example, it shouldn't fail as such? I checked it with `lychee` locally and it works as usual

Reply:
But why even bother with a link when we can just use words

Reply:
Makes sense, could you an extended description? Like not to be excessively wordy, but just another sentence about what additional context people can provide besides their snippets in code blocks